go/engine/buffered_identify_ui.go

```diff
@@ -224,6 +225,14 @@ func (b *bufferedIdentifyUI) ReportTrackToken(t keybase1.TrackToken) error {
 	return b.raw.ReportTrackToken(t)
 }
 
+func (b *bufferedIdentifyUI) Cancel() error {
+	b.Lock()
+	defer b.Unlock()
+
+	// Cancel should always go through to UI server
+	return b.raw.Cancel()
```

Comment:
This won't wind up being called in `Identify2WithUID`.  I think that's fine but just double-checking with you.